pages/learn/glossary.md

```diff
@@ -106,6 +112,13 @@ Strictly speaking, according to the specification, schemas are themselves [JSON
 
 In recent [drafts](#draft) of the specification, a schema is either a JSON object or a JSON boolean value.
 
+#### schema resource
+
+A schema resource is a [schema](#schema) which is canonically identified by an absolute URI. This implies all root/parent schemas are schema resources as they can be identified by an absolute URI. 
```

Comment:
> A schema resource is a [schema](#schema) which is canonically identified by an absolute URI.

I'd drop the word "canonically". It's just techno-babel that's not necessary to describe the concept to lay-people.